eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -1,17 +1,29 @@
 # Note, due to how `Expand-Archive` is leveraged in this script,
 # powershell core is a requirement for successful execution.
 param (
+  # used by VerifyPackages
+  $artifactLocation, # the root of the artifact folder. DevOps $(System.ArtifactsDirectory)
+  $workingDirectory, # directory that package artifacts will be extracted into for examination (if necessary)
+  $packageRepository, # used to indicate destination against which we will check the existing version.
+  # valid options: PyPI, Nuget, NPM, Maven, C, CPP
+  # used by CreateTags
+  $releaseSha, # the SHA for the artifacts. DevOps: $(Release.Artifacts.<artifactAlias>.SourceVersion) or $(Build.SourceVersion)
+  [switch]$continueOnError = $true,
+
   $AzCopy,
   $DocLocation,
   $SASKey,
   $Language,
   $BlobName,
   $ExitOnError=1,
-  $UploadLatest=1
+  $UploadLatest=1,
+  $RepoReplaceRegex
 )
+. (Join-Path $PSScriptRoot link-replacement.ps1)
+. (Join-Path $PSScriptRoot artifact-metadata-parsing.ps1)
 
 $Language = $Language.ToLower()
-
+$DocDefaultName = 'docs'
```

Comment:
This is not a good idea we don't want to hard-code this and it doesn't scale. We should always have the package info.